lib/greengrasscoreipc/model.ts

```diff
@@ -447,20 +671,32 @@ export interface ComponentNotFoundError {
 
 export interface StopComponentResponse {
 
+    /**
+     * The status of the stop request.
+     */
     stopStatus: RequestStatus,
 
+    /**
+     * A message about why the component failed to stop, if the request failed.
+     */
     message?: string
 
 }
 
 export interface StopComponentRequest {
 
+    /**
+     * The name of the component.
```

Comment:
Very Trivial: The other descriptions list what is being asked. e.g. "The name of the component to pause". Should this follow and be "The name of the component to stop."?